macros/upload_models.sql

```diff
@@ -61,7 +65,9 @@
                     '{{ model.package_name }}', {# package_name #}
                     '{{ model.original_file_path | replace('\\', '\\\\') }}', {# path #}
                     '{{ model.checksum.checksum }}', {# checksum #}
-                    '{{ model.config.materialized }}' {# materialization #}
+                    '{{ model.config.materialized }}', {# materialization #}
+                    '{{ tojson(model.tags) }}', {# tags #}
```

Comment:
```suggestion
                    {{ tojson(model.tags) }}, {# tags #}
```

Just removing the quotes here so that it passes on BigQuery.